pkgs/development/libraries/science/chemistry/openmm/default.nix

```diff
@@ -6,10 +6,16 @@
 , fftwSinglePrec
 , doxygen
 , swig
-, python3Packages, enablePython ? false
-, opencl-headers, ocl-icd, enableOpencl ? false
-, clang, enableClang ? true
```

Comment:
`clang` should be provided via stdenv? Would it work if we put remove the switch and decide further down with `stdenv.cc.isClang` if we are using clang?

Reply:
Hm I would assume so. We already had some discussion in the init of OpenMM regarding CLang: https://github.com/NixOS/nixpkgs/pull/188970#discussion_r959356443

I think it will be completely fine to use an override of the stdenv if Clang is required. I will just remove the clang stuff.

Reply:
It think this cleaner (given it works). Now, we are mixing gcc and clang into the build.